lib/ReactViews/ExplorerWindow/Tabs/DataCatalogTab.tsx

```diff
@@ -0,0 +1,92 @@
+import { runInAction } from "mobx";
+import { observer } from "mobx-react";
+import { useTranslation } from "react-i18next";
+import type CatalogMemberMixin from "../../../ModelMixins/CatalogMemberMixin";
+import Box from "../../../Styled/Box";
+import { useViewState } from "../../Context";
+import DataCatalog from "../../DataCatalog/DataCatalog";
+import DataPreview from "../../Preview/DataPreview";
+import Breadcrumbs from "../../Search/Breadcrumbs";
+import SearchBox, { DEBOUNCE_INTERVAL } from "../../Search/SearchBox.jsx";
+import Styles from "./data-catalog-tab.scss";
+
+interface DataCatalogTabProps {
+  items?: unknown[];
+  searchPlaceholder?: string;
+  onActionButtonClicked?: (item: CatalogMemberMixin.Instance) => void;
+}
+
+const DataCatalogTab = observer(function DataCatalogTab(
```

Comment:
The `const DataCatalogTab` gets shadowed by the function name inside `observer()`, is that intentional or could the function passed to the observer be without name?